src/main/frontend/src/code-quarkus/pickers/extensions-picker.tsx

```diff
@@ -311,7 +338,7 @@ export const ExtensionsPicker = (props: ExtensionsPickerProps) => {
             fieldId="search-extensions-input"
           >
             <SearchIcon/>
-            <TextInput
```

Comment:
Fixed!

What I did was remove the useDebounce because it was not affecting the filter from the TextInput, so I used the useRef with the lodash debounce, to debounce the function instead. And another thing is that I put 1000ms (1 second) because I tested and I think it is good time to change the params.

I changed the snap file again in order to rollback the altering of TextInput.